pkg/apis/componentconfig/cloudcore/v1alpha1/types.go

```diff
@@ -250,6 +258,8 @@ type EdgeControllerBuffer struct {
 type ControllerContext struct {
 	// SendModule indicates which module will send message to
 	SendModule metaconfig.ModuleName `json:"sendModule,omitempty"`
+	// SendRouterModule indicates messages will send message to router module.
```

Comment:
`SendRouterModule indicates messages will send message to router module`  not smoothly: messages will send message?

Reply:
done